tests/system/marathon_common_tests.py

```diff
@@ -826,14 +826,14 @@ def test_app_update_rollback():
 
     client = marathon.create_client()
     client.add_app(app_def)
-    shakedown.deployment_wait()
+    shakedown.deployment_wait(app_id=app_id)
```

Comment:
I would not conflate these changes with the subrepo change

Reply:
Yep. You are right. I won't merge this until https://github.com/mesosphere/marathon/pull/6302 landed.